.github/pull_request_template.md

```diff
@@ -8,6 +8,10 @@ Fixes #[insert GH or internal issue number(s)].
   - [ ] TODO (before merge)
   - [ ] Included
   - [ ] Not necessary because:
+- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
+  - [ ] I don't know
+  - [ ] Required / Maybe required
+  - [ ] Not required because:
```

Comment:
This is useful to confirm there is no need for the e2e test run. And to make sure they weren't run only because we forgot to add the label.